tests/system/providers/amazon/aws/example_eks_with_fargate_in_one_step.py

```diff
@@ -81,6 +81,9 @@
         # Opting to use the same ARN for the cluster and the pod here,
         # but a different ARN could be configured and passed if desired.
         fargate_pod_execution_role_arn=fargate_pod_role_arn,
+        deferrable=True,
+        waiter_delay=30,
+        wait_for_completion=399,
```

Comment:
This doesn't appear related to the rest of the PR.

Also: System tests don't currently support deferrable operators, and wait_for_completion is a bool.